### PR TITLE
DEV-2632 | Improve copy in Connect to Stripe modal and toast message to clarify what Stripe is and when account creation happens

### PR DESCRIPTION
**Pull request by benjaminEwhite**
Please fill out each section even if it's just with "N/A"

#### What's this PR do?

This PR updates copy and tweaks styling on the Stripe Connect modal and toast components.

The ticket requires that the "Need help? Check out our FAQ" copy be added to the connect toast component, so I moved that into a separate component that can be shared by both `ConnectStripeModal` and `ConnectStripeToast`.

#### Why are we doing this? How does it help us?

Make the why and how of Stripe connection more obvious to new users 

#### How should this be manually tested? Please include detailed step-by-step instructions.

Go through the "new account" flow in the review app.

When first presented with the `ConnectStripeModal`, compare to requirements from ticket. 

Click "I'll connect to Stripe later" to hide modal and cause the `ConnectStripeToast` to appear. 

Inspect the toast component and compare to requirements from ticket.

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?

No

#### Have automated unit tests been added? If not, why?

No. This is just copy changes and styling tweaks, so didn't seem necessary. 

#### How should this change be communicated to end users?

N/A

#### Are there any smells or added technical debt to note?

No

#### Has this been documented? If so, where?

N/A

#### What are the relevant tickets? Add a link to any relevant ones.

[DEV-2632](https://news-revenue-hub.atlassian.net/browse/DEV-2632)

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?

No

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:

N/A

### CODE REVIEW

File: spa/src/components/dashboard/connectStripe/ConnectStripeNeedHelpCTA.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import * as S from './ConnectStripeNeedHelpCTA.styled';
```
Comment (nrh-cklimas) [inline spa/src/components/dashboard/connectStripe/ConnectStripeNeedHelpCTA.tsx:1]: Destructure this import?

**Review by nrh-cklimas — CHANGES_REQUESTED** [review]

File: spa/src/components/dashboard/connectStripe/ConnectStripeNeedHelpCTA.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <S.NeedHelpCta>
+      <S.NeedHelpSpan>Need help?</S.NeedHelpSpan>
+      <S.CheckOutOur>Check out our</S.CheckOutOur>
+      <S.StripeFAQ href={CONNECT_STRIPE_FAQ_LINK} target="_blank">
+        FAQ
+      </S.StripeFAQ>
+      .
+    </S.NeedHelpCta>
+  );
+}
```
Comment (nrh-cklimas) [inline spa/src/components/dashboard/connectStripe/ConnectStripeNeedHelpCTA.tsx:15]: We should add tests for this component, as simple as it is. I'd go with a simple "is accessible" check and also test that the link is present and points to the correct URL.

File: spa/src/components/dashboard/connectStripe/ConnectStripeNeedHelpCTA.styled.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+export const NeedHelpCta = styled.div`
+  margin: 17px 0px 4px;
+`;
+
+export const NeedHelpSpan = styled.span`
+  font-weight: 500;
+  margin-right: 0.75em;
+`;
+
+export const CheckOutOur = styled.span`
```
[the remark this answers is not in the document]
Comment (benjaminEwhite) [reply spa/src/components/dashboard/connectStripe/ConnectStripeNeedHelpCTA.styled.ts]: aa2d3a7e71e

File: spa/src/components/dashboard/connectStripe/ConnectStripeNeedHelpCTA.tsx (hunk 1 above)
Comment (benjaminEwhite) [reply spa/src/components/dashboard/connectStripe/ConnectStripeNeedHelpCTA.tsx]: 40bb2efd8bf 

Ended up removing the `* as S` from the toast `ConnectStripeElements.js` and `ConnectStripeToast` as well.

File: spa/src/components/dashboard/connectStripe/ConnectStripeNeedHelpCTA.tsx (hunk 2 above)
Comment (benjaminEwhite) [reply spa/src/components/dashboard/connectStripe/ConnectStripeNeedHelpCTA.tsx:14]: e302c5bdc24

Comment (benjaminEwhite) [reply spa/src/components/dashboard/connectStripe/ConnectStripeNeedHelpCTA.tsx:14]: e302c5bdc24

**Review by x110dc — APPROVED** [review]

**Pull request merged**